README.md

```diff
@@ -3,263 +3,441 @@
+
+
+
+## Paths
+
+### <span id="create-subscription"></span> Creates a subscription resource for the Event Consumer. (*createSubscription*)
 
 ```
-# Pub/Sub Rest API
-Rest API spec .
+POST /api/ocloudNotifications/v2/subscriptions
+```
 
-## Version: 1.0.0
+Creates a new subscription for the required event by passing the appropriate payload.
 
+#### Parameters
 
-### /publishers/
+| Name | Source | Type | Go type | Separator | Required | Default | Description |
+|------|--------|------|---------|-----------| :------: |---------|-------------|
+| SubscriptionInfo | `body` | [SubscriptionInfo](#subscription-info) | `models.SubscriptionInfo` | |  | | The payload will include an event notification request, endpointUri and ResourceAddress. The SubscriptionId and UriLocation are ignored in the POST body (these will be sent to the client after the resource is created). |
 
-#### POST
-##### Summary
+#### All responses
+| Code | Status | Description | Has headers | Schema |
+|------|--------|-------------|:-----------:|--------|
+| [201](#create-subscription-201) | Created | Shall be returned when the subscription resource is created successfully. |  | [schema](#create-subscription-201-schema) |
+| [400](#create-subscription-400) | Bad Request | Bad request. For example, the endpoint URI is not correctly formatted. |  | [schema](#create-subscription-400-schema) |
+| [404](#create-subscription-404) | Not Found | Not Found. Subscription resource is not available. For example, PTP is not supported by the node. |  | [schema](#create-subscription-404-schema) |
+| [409](#create-subscription-409) | Conflict | Conflict. The subscription resource already exists. |  | [schema](#create-subscription-409-schema) |
 
-Creates a new publisher.
+#### Responses
 
-##### Description
 
-If publisher creation is success(or if already exists), publisher will be returned with Created (201).
+##### <span id="create-subscription-201"></span> 201 - Shall be returned when the subscription resource is created successfully.
+Status: Created
 
-##### Parameters
+###### <span id="create-subscription-201-schema"></span> Schema
 
-| Name | Located in | Description | Required | Schema |
-| ---- | ---------- | ----------- | -------- | ---- |
-| publisher | body | publisher to add to the list of pub | [PubSub](#pubsub) |
 
-##### Responses
 
-| Code | Description | Schema |
-| ---- | ----------- | ------ |
-| 201 | publisher/subscription data model | [PubSub](#pubsub) |
-| 400 | Error Bad Request | object |
+[SubscriptionInfo](#subscription-info)
 
-### /subscriptions/
+##### <span id="create-subscription-400"></span> 400 - Bad request. For example, the endpoint URI is not correctly formatted.
+Status: Bad Request
 
-#### POST
-##### Summary
+###### <span id="create-subscription-400-schema"></span> Schema
 
-Creates a new subscription.
+##### <span id="create-subscription-404"></span> 404 - Not Found. Subscription resource is not available. For example, PTP is not supported by the node.
```

Comment:
same -- subscription  for the resource not avaiable

Reply:
removed PTP example